### ISSUE TITLE
Grunt + browserify + nunjucks

### ISSUE THREAD

**Issue by Nachbarshund** [mozilla/nunjucks#669]
Since version `2.x` there is a problem to compile and use nunjucks template on client and serverside.

Here my configuration

Package.json:
```json
{
  "name": "grunt-nunjucks-browserfiy",
  "version": "1.0.0",
  "description": "",
  "devDependencies": {
    "browserify-shim": "3.8.12",
    "grunt": "0.4.5",
    "grunt-browserify": "4.0.1",
    "nunjucksify": "0.x"
  },
  "dependencies": {
    "coffeeify": "2.0.1",
    "nunjucks": "2.x"
  },
  "browser": {
    "nunjucks": "nunjucks/browser/nunjucks-slim"
  }
}
```

Gruntfile.coffee:
```coffee
module.exports = (grunt) ->
	
	# Project configuration.
	grunt.initConfig		
		browserify:
			main:
				files:
					'main.nun.js': "main.coffee"
				options:
					transform: [
						"coffeeify",
						["nunjucksify", { extension: '.html' } ]
					]
					browserifyOptions:

						debug: true
						extensions: ".coffee"

	# Load npm modules
	grunt.loadNpmTasks "grunt-browserify"
```

main.coffee:
```coffee
tmplMng = require( "./templates" )
view = tmplMng( require( "./views/test.html" ) ) 


console.log "main.coffee", view( {} )
````
index.html:
```html
<html>
	<head>
		<title>Nunjucks Browserify Test</title>
	</head>
	<body>
		<h1>Nunjucks Browserify Test</h1>
		<script src="main.nun.js"></script>
	</body>
</html>
```

### Result
```javascript
Uncaught TypeError: nunjucks.Environment is not a function
```

The latest working version: `1.3.4`

Is there any solution? I can't use the version `1.3.4` because on serverside there is not the options to use the `getGlobal()` function, e.g. `nunjucksEnv.getGlobal()`

**Comment by oyyd** [comment]
Maybe you need to exports `nunjucks` object manually in 2.x. Something like: 
```js
window.nunjucks = require('nunjucks');
```

**Comment by carljm** [comment]
I don't use grunt or browserify myself, so I'm not sure how helpful I can be here with workarounds, but this is almost certainly a symptom of #507, and I believe it would probably be fixed by the plan I've outlined in #507.

**Comment by carljm** [comment]
Closing this as a duplicate of #507.
